specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2021-03-08/workbookOperations_API.json

```diff
@@ -0,0 +1,158 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "WorkbookClient",
+    "description": "Azure client for Workbook.",
+    "version": "2021-03-08"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow.",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/providers/Microsoft.Insights/operations": {
```

Comment:
This URL is for operations of ai. It's not only for workbook Operations. The supports api-versions are  '2014-04-01,2014-06-01,2015-04-01'.
![image](https://user-images.githubusercontent.com/69238381/115979179-05cf4500-a5b7-11eb-9bf3-090300df1303.png)

Reply:
the Microsoft.Insights RP is huge and is maintained by many teams. The operations api does not include the sub apis' api-version. So you can just use the 2015-04-01 api-version. 

The newly added endpoint "/Workbooks/Revisions/Read" was merged into the RP yesterday: https://msazure.visualstudio.com/One/_git/AzureUX-Insights-MonRP/pullrequest/4386680